asserts/assertstest/assertstest.go

```diff
@@ -237,7 +237,9 @@ func NewSigningDB(authorityID string, privKey asserts.PrivateKey) *SigningDB {
 }
 
 func (db *SigningDB) Sign(assertType *asserts.AssertionType, headers map[string]interface{}, body []byte, keyID string) (asserts.Assertion, error) {
-	headers["authority-id"] = db.AuthorityID
+	if _, ok := headers["authority-id"]; !ok {
```

Comment:
I don't think the arg and the map entry should behave the same, but maybe it's me